content/docs/pipelines/overview/pipelines-overview.md

```diff
@@ -33,9 +33,10 @@ The following are the goals of Kubeflow Pipelines:
 * Easy re-use: enabling you to re-use components and pipelines to quickly 
   create end-to-end solutions without having to rebuild each time.
 
-In 
-[Kubeflow v0.1.3 and later](https://github.com/kubeflow/pipelines/releases/tag/0.1.3),
-Kubeflow Pipelines is one of the Kubeflow core components. It's automatically deployed during Kubeflow deployment.
+Kubeflow Pipelines is available as a standalone installation and as a core Kubeflow component.
```

Comment:
Suggested revision: Focuses on KFP as a component of Kubeflow, and updates installation options to be a list.

Kubeflow Pipelines is available as a core component of Kubeflow or as a standalone installation.

* Learn more about [installing Kubeflow](/docs/started/getting-started/).
* Learn more about [installing Kubeflow Pipelines standalone](/docs/pipelines/installation/overview/).

Reply:
👍